tlslite/tlsrecordlayer.py

```diff
@@ -147,6 +147,9 @@ def __init__(self, sock):
         #Limit the size of outgoing records to following size
         self.recordSize = 16384 # 2**14
 
+        # Indicator for heartbeat extension
+        self.heartbeat = False
```

Comment:
what if we can't send requests but server can? I think one variable is not enough to keep the state